### DISCUSSION TITLE
Nextcloud AIO - external storage

### DISCUSSION THREAD

**Discussion by wisdomlight** [nextcloud/all-in-one#3675] [Questions] [answered]
![Nextcloud Warning Screenshot from 2023-11-04 21-20-46](https://github.com/nextcloud/all-in-one/assets/14957169/896ad632-f32a-4e52-a7d0-2d022f2cf433)

![Nextcloud Disks Screenshot from 2023-11-04 21-03-32](https://github.com/nextcloud/all-in-one/assets/14957169/a55c8d31-d697-48cb-9d6d-c6213e5d3bd2)
After years of NCP I tried AIO and looks impressive. Thank you.
I installed using this command (with additional line for external storage as recommended by documentation)
```
sudo docker run \
--init \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 80:80 \
--publish 8080:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
-e NEXTCLOUD_DATADIR=/media/ncdata/nextcloud \
nextcloud/all-in-one:latest
```
Looking at the external storage - indeed this is where everything is stored.
### The problem
edit
How do I move the datadir from `/media/ncdata/nextcloud/` to `/media/user/ncdata/nextcloud/`
is it a simple `mv /media/ncdata/nextcloud /media/user/ncdata/nextcloud/` and then `docker exec -u www-data php occ files:scan --all` ?

or do I need to `docker exec nextcloud-aio-nextcloud -it /bin/bash` then `cd /mnt/ncdata/` and somehow copy it to the external storage??

p.s
only after I posted this post I realized the mistake I made - and so my original question became obselet.
It does not seem possible to delete questions or change the question heading

**Comment by szaimen** [comment, marked as answer]
> only after I posted this post I realized the mistake I made - and so my original question became obselet.
> It does not seem possible to delete questions or change the question heading
